app/src/androidTest/java/mozilla/lockbox/AppRoutePresenterTest.kt

```diff
@@ -93,8 +96,11 @@ open class AppRoutePresenterTest {
         navigator.gotoDisconnectDisclaimer()
         navigator.back()
         navigator.checkAtAccountSetting()
+        navigator.back()
+        navigator.back()
```

Comment:
It is like for the simulator on Firebase we need to go back from the Disconnect message and from Account settings so that the next tests work...strange :(